types/raygun/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{ 
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false
```

Comment:
Please don't turn off linter rules, instead fix the warnings.

Reply:
Thanks for your comment, but typescript doesn't like top level don't have declare.
![screen shot 2019-02-19 at 3 01 25 pm](https://user-images.githubusercontent.com/3240167/52989393-48d18300-3457-11e9-9e76-a5b5d766ede3.png)

Reply:
You're right - @sandersn any idea the intention is with this linter rule? I don't really understand why the rule passed before the `export { Client }` change.

```
ERROR: 7:1    strict-export-declare-modifiers  'declare' keyword is redundant here. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 123:1  strict-export-declare-modifiers  'declare' keyword is redundant here. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
```

Reply:
Well, there's an explicit exemption in the rule for `export = Client` syntax. I'm not sure why, perhaps because of the old practise of using global declarations along with export assignments to make UMD. 

As far as I can tell, the correct syntax is to add `export` before `namespace` instead of `declare`. I'll double-check that.

Reply:
This is the result of two bad error messages:

1. The error from the says 'declare' is required, but 'export' also works.
2. The error message from the lint rule is incorrect; it should say that 'export' is preferred, not that 'declare' is redundant. I'm not sure if the message is always wrong or just in this case. Note that a comment in the lint rule's source is correct:

```ts
// `declare global` and `declare module "foo"` OK. `declare namespace N` not OK, should be `export namespace`.
```

I'll open a bug on dtslint to improve the error message.

Reply:
@sandersn`strict-export-declare-modifiers` complains at `declare class` as well at line 123. What to do with it?

Reply:
After further discussion with our local experts on `export` and `declare`, I fixed the rule; `declare` is actually required here and the rule is wrong. I'll probably merge and ship the fix tomorrow:
https://github.com/Microsoft/dtslint/pull/206

Reply:
@sandersn Has that been deployed? I tried reinstall dtslint and run `npm run lint raygun`, but the error is still there. I'll open a new PR once this is fixed.